pkg/common/resource.go

```diff
@@ -139,6 +139,9 @@ func (r *Resource) SetCapacityToNode(node *corev1.Node) {
 		corev1.ResourceEphemeralStorage: empStorage,
 	}
 	for name, quota := range r.Custom {
+		if quota.Sign() < 0 {
+			continue
+		}
```

Comment:
I am not sure if add this logic will lead to some unexpected issues, e.g. `evicting pods`. Can you add some test for these scenes?

Reply:
or, we can simply set the quantity to 0? since kubernetes cannot accept negtive values.